lib/Minz/ModelPdo.php

```diff
@@ -6,22 +6,27 @@
  */
 
 /**
- * La classe Model_sql représente le modèle interragissant avec les bases de données
+ * La classe Model_sql représente le modèle interagissant avec les bases de données
```

Comment:
If we're changing the line anyway, should we change it to English?

`The Model_sql class represents the database interaction model`, or something along those lines.

Reply:
Done